pulsar-functions/utils/src/test/java/org/apache/pulsar/functions/utils/SinkConfigUtilsTest.java

```diff
@@ -390,22 +390,6 @@ public void testMergeRuntimeFlags() {
         );
     }
 
-    @Test
-    public void testMergeDifferentCleanupSubscription() {
```

Comment:
Could you please rebase the master branch? I think the test has already been removed on the master branch.

Reply:
the method still exists yesterday, but I checkout just now, it removed.